README.md

```diff
@@ -46,11 +66,41 @@ or
 
 ```
 pip install git+https://github.com/The-AI-Alliance/gofannon.git
+# OR
+pip install gofannon
+```
+
+## Communication Channels
+- **Discord** [Join our Discord server](https://discord.gg/2MMCVs76Sr)for eal-time collaboration and support
+- **GitHub Discussions**: Explore our [GitHub organization](https://github.com/The-AI-Alliance/agents-wg/discussions/) for all related projects
+- **Community Calls**: [Join our bi-weekly video meetings](https://calendar.app.google/c4eKW4zrNiXaue926)
+
+## Usage Example
+```bash
+from gofannon.base import BaseTool  
+  
+class NewTool(BaseTool):  
+    def __init__(self):  
+        super().__init__()  
+  
+    @property  
+    def definition(self):  
+        return {  
+            # Define your tool metadata and configuration  
+        }  
+  
+    def fn(self, *args, **kwargs):  
+        # Define your tool functionality  
+        pass  
```

Comment:
I think a better example is possible. But that's not on you, this isn't really an example, and any of the other examples don't really show it off doing anything. This is all a long winded way of saying, we need to come up with a good example for the readme, and that should be a follow on issue. Not blocking on this PR